tigerpath/majors_and_certificates/majors/HIS.json

```diff
@@ -32,14 +33,14 @@
               "name":"Prerequisites",
               "max_counted":1,
               "min_needed": 1,
-              "explanation":"Students are required to take and pass at least two departmental courses before they enter the department. Students who wish to enter the department but who have not taken two departmental courses before their junior year must consult with the departmental representative.",
+              "explanation":"Before they enter the department, students in the class of 2023 are required to take and pass one (1) 200-level course, from the list designated as prerequisites.",
               "completed_by_semester":4,
               "double_counting_allowed":true,
               "req_list":[
                 {
                   "name": null,
                   "max_counted":1,
-                  "min_needed": 2,
+                  "min_needed": 1,

```

Comment:
"Before they enter the department, students in the class of 2023 are required to take and pass one (1) 200-level course, from the list designated as prerequisites."